satellite/satellitedb/users.go

```diff
@@ -143,3 +145,7 @@ func userFromDBX(ctx context.Context, user *dbx.User) (_ *console.User, err erro
 
 	return &result, nil
 }
+
+func normalizeEmail(email string) string {
+	return strings.ToUpper(email)
```

Comment:
Please verify that the SQL `UPPER` behaves the same as `strings.ToUpper`

Reply:
Added few tests